extensions/nbnhhsh/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nbnhhsh",
+  "title": "nbnhhsh",
+  "description": "nbnhhsh raycast script",
```

Comment:
Could you describe this extension so people see it and understand what it can do?